README.md

```diff
@@ -22,7 +22,11 @@ Or install it yourself as:
 
 ```ruby
 
-summary_document = TelephoneAppointments::SummaryDocumentActivity.new(appointment_id: id, owner_uid: uid)
+summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+  appointment_id: id,
+  owner_uid: uid, 
+  message: message # 'postal' or 'digital'
```

Comment:
I went with "message" as it matches the fields on the TAP side and as such so would any error messaging.